topologic/io/csv_loader.py

```diff
@@ -40,37 +31,28 @@ def from_dataset(
       edge or vertex in a list of metadata dictionaries
 
     You can certainly provide your own projection function generators for specialized needs; just ensure they follow
-    the type signature of Callable[[networkx.Graph, MetadataTypeRegistry], Callable[[List[str]], None]]
-
-    Note that even if you don't want any metadata to be retained, we still require a MetadataTypeRegistry to be
-    configured for each 2nd layer function configuration.  Ensure you follow the type signatures referenced even if you
-    ignore the registry for your own needs.
+    the type signature of Callable[[networkx.Graph], Callable[[List[str]], None]]

```

Comment:
We are mixing networkx.Graph and nx.Graph between docs. Most of the other docs in the project are using the nx alias.